src/main/scala/edu/ie3/util/quantities/QuantityUtils.scala

```diff
@@ -211,19 +213,35 @@ object QuantityUtils {
     def asMicroFarradPerKilometre: ComparableQuantity[SpecificCapacitance] =
       as(MICROFARAD_PER_KILOMETRE)
 
-    def asKiloWattHourPerKelvin: ComparableQuantity[HeatCapacity] =
-      as(KILOWATTHOUR_PER_KELVIN)
-
     def asKiloWattHourPerKelvinTimesCubicMetre
```

Comment:
Sorry for being pedantic, but doesn't this one belong to the thermal category as well?